lib/bom.js

```diff
@@ -682,14 +682,15 @@ type PositionOptions = {
     maximumAge?: number;
 }
 
-declare class AudioContext {
+type AudioContextState = 'suspended' | 'running' | 'closed';
+
+declare class BaseAudioContext {
   currentTime: number;
   destination: AudioDestinationNode;
   listener: AudioListener;
   sampleRate: number;
-  state: any;
```

Comment:
`state` is one of `suspended`, `running` and `closed`.
https://webaudio.github.io/web-audio-api/#enumdef-audiocontextstate